execution/exec_options_table.ml

```diff
@@ -48,7 +48,6 @@ let  getParameterTable () =
     add_parameter "-solver-check-against" (String OS.opt_solver_check_against);
     add_parameter "-solver-path" (String EO.opt_solver_path);
     add_parameter "-smtlib-solver-type" (StringOpt OS.opt_smtlib_solver_type_string);
-    add_parameter "-save-solver-files" (Bool EO.opt_save_solver_files);
```

Comment:
Why are you removing this from the table?

Reply:
I believe this line is removed by mistake. I'll add anothe commit to undo it.